HW_2/task_2.py

```diff
@@ -0,0 +1,42 @@
+# Была выполнена 2 задача из домашнего задания номер 2.
+
+# 2. Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем и знаменателем.
+# Программа должна возвращать сумму и произведение* дробей.
+# Для проверки своего кода используйте модуль fractions.   
+
+
+# from fractions import Fraction
+import fractions
+
+frac1  = input('Введи первую дробь в формате “a/b”: ')
+frac2  = input('Введи вторую дробь в формате “a/b”: ')
+# frac1  = "1/3"
+# frac2  = "3/5"
+
+divider1 = int(frac1[:frac1.find("/")])
+denominator1 = int(frac1[frac1.find("/")+1:])
+divider2 = int(frac2[:frac2.find("/")])
+denominator2 = int(frac2[frac2.find("/")+1:])
```

Comment:
На мой взгляд, так проще
```suggestion
divider1, denominator1  = map(int, frac1.split("/"))
divider2, denominator2  = map(int, frac2.split("/"))
```